CMakeLists.txt

```diff
@@ -182,7 +182,7 @@ if(NOT AUTOWIRING_IS_EMBEDDED)
   install(FILES
     "${CMAKE_CURRENT_BINARY_DIR}/autowiring-config.cmake"
     "${CMAKE_CURRENT_BINARY_DIR}/autowiring-configVersion.cmake"
-    DESTINATION "cmake"
+    DESTINATION ${CMAKE_INSTALL_PREFIX}
```

Comment:
No. ${CMAKE_INSTALL_PREFIX} is absolute path (default to /usr/local). "." is relative path, the config files will be installed to ${CMAKE_INSTALL_PREFIX}/./leapserial-config.cmake and ${CMAKE_INSTALL_PREFIX}/./leapserial-configVersion.cmake.